core/src/main/java/hudson/ProxyConfiguration.java

```diff
@@ -333,7 +333,7 @@ private static ProxyConfiguration _get() {
         JenkinsJVM.checkJenkinsJVM();
         // this code could be called between the JVM flag being set and theInstance initialized
         Jenkins jenkins = Jenkins.get();
-        return jenkins == null ? null : jenkins.proxy;
```

Comment:
I am not sure this is right taking the comment above. maybe it should be `getInstanceOrNull` (and we made a mistake during previous refactorings)

Reply:
I did change it to getInstanceOrNull() as suggested.